piff/input.py

```diff
@@ -556,6 +569,16 @@ def _makeStarsFromImage(image_kwargs, cat_kwargs, wcs, chipnum,
 
         nstars_in_image = 0
         stars = []
+
+        if max_edge_frac is not None:
+            edge_radius = 5./12.                # fraction of stamp_size to use as a radius
```

Comment:
This value still needs to be documented to the user somewhere (i.e. not just an inline code comment).  I think the easiest would be to make it a user-settable parameter, since that provides an obvious place to explain what it means.  But if not that, then either in the explanation of the `max_edge_frac` parameter or in the main section of the doc string, where you could give a long-form (paragraph) explanation of how `max_edge_frac` works.